errors/errors.go

```diff
@@ -51,6 +51,9 @@ const (
 	UnsupportedContact
 	// The requesteed serial number does not exist in the `serials` table.
 	UnknownSerial
+	// The certificate being indicated for replacement already has a replacement
```

Comment:
To me, this reads like we could instead re-use `berrors.Duplicate`.

Reply:
Because only one identifier matches, we can't say with any certainty that it is actually a duplicate. That's why we decided on conflict instead.